src/library_wasmfs.js

```diff
@@ -25,6 +25,7 @@ FS.createPreloadedFile = FS_createPreloadedFile;
 #if FORCE_FILESYSTEM
     '$FS_modeStringToFlags',
     'malloc',
+    'free',
```

Comment:
It looks like you removed the call to _free below but do you still want to add this dependnecy?

Reply:
This is kind of a drive-by fix. We need this dep in force-filesystem mode.